### PR TITLE
ipatests: add check for output contents of ipa-client-samba

### PR DESCRIPTION
**Pull request by wladich**
Check that ipa-client-samba  tool reports specific properties of domains:
name, netbios name, sid and id range

Related to https://pagure.io/freeipa/issue/8149

### CODE REVIEW

File: ipatests/test_integration/test_smb.py [hunk 1]
```diff
@@ -194,6 +195,31 @@ def smb_sanity_check(self, user, client_mountpoint, share):
         finally:
             run_smb_server(['rm', '-rf', test_dir], raiseonerr=False)
 
+    def smb_installation_check(self, result):
+        domain_regexp_tpl = r'''
+            Domain\ name:\s*{domain}\n
+            \s*NetBIOS\ name:\s*{netbios}\n
+            \s*SID:\s+S-1-5-\d+-\d+-\d+-\d+\n
```
Comment (abbra) [inline ipatests/test_integration/test_smb.py]: Please check for `S-1-5-21-<domain>` here. (21 is important).

Comment (wladich) [reply ipatests/test_integration/test_smb.py]: fixed

**Comment by wladich** [comment]
@abbra Please review

**Comment by abbra** [comment]
LGTM. ACK.

**Comment by wladich** [comment]
Removed rtemp commit, waiting for tests

**Comment by flo-renaud** [comment]
master:

* 15fd36612ea451cec2e1302a462fa440663fcc74 ipatests: add check for output contents of ipa-client-samba

**Pull request closed without merging**